openshift/templates/api/api-deploy.bc.prod.param

```diff
@@ -33,7 +32,7 @@
 # RTI_MAX_SOLR_BATCH=25
 # RTI_RAISE_ERRORS=false
 # RTI_ABORT_ON_ERRORS=true
-TAG_NAME=bc-prod
+TAG_NAME=prod
```

Comment:
@WadeBarnes just confirming with you we can now just use "simple" tags for images since we have a single instance.

Reply:
Confirmed this is correct